docs/reference/glossary.md

```diff
@@ -0,0 +1,7 @@
+# Glossary
+
+<!-- This is a work in progress. It is not added to the navigation yet, but exists to begin documenting terms. -->
+
+## Document set
```

Comment:
We need some word for it. We can change this later if we have something better, at which point we can also replace it in the text.